include/mg_procedure.h

```diff
@@ -164,6 +164,9 @@ enum mgp_value_type {
   MGP_VALUE_TYPE_DURATION,
 };
 
+/// All available log levels that can be use in mgp_log function
+enum mgp_log_level { Trace, Debug, Info, Warn, Error, Critical };
```

Comment:
```suggestion
MGP_ENUM_CLASS mgp_log_level {
  TRACE,
  DEBUG,
  INFO,
  WARN,
  ERROR,
  CRITICAL,
};
```

Use `enum class` in C++ and use uppercase enum names. In C the single `PascalCase` enum values can be misleading.

Reply:
Changed it 👍🏻